src/arch/riscv/insts/vector.cc

```diff
@@ -429,16 +468,16 @@ VMaskMergeMicroInst::VMaskMergeMicroInst(ExtMachInst extMachInst,
 
     setDestRegIdx(_numDestRegs++, vecRegClass[_dstReg]);
     _numTypedDestRegs[VecRegClass]++;
-    for (uint8_t i=0; i<_numSrcs; i++) {
+    for (uint8_t i = 0; i < _numSrcs; i++) {
         setSrcRegIdx(_numSrcRegs++, vecRegClass[VecMemInternalReg0 + i]);
     }
 }
 
 Fault
-VMaskMergeMicroInst::execute(ExecContext* xc,
-    trace::InstRecord* traceData) const
+VMaskMergeMicroInst::execute(ExecContext *xc,
+                             trace::InstRecord *traceData) const
```

Comment:
Is there a good argument against this style? I fear we're going to go back and forth forever on stuff like this. I personally don't see much wrong with it.

Reply:
Sorry for lately reply. I have hide and deleted some commit not required update. 

> Is there a good argument against this style? I fear we're going to go back and forth forever on stuff like this. I personally don't see much wrong with it.

Some of format option required are updated here: https://github.com/gem5/gem5/pull/362#discussion_r1413750610. 

For the part of MicroStaticInst implement. I feel it will be little harder to compare the StaticInst generated from `isa_parser.py`.

Feel free to leave comments, thanks.